docs/timelines/creating-a-timeline.md

```diff
@@ -0,0 +1,103 @@
+
+- While you are in Live Record, the console will follow the Timeline Release settings as you've programmed them in the [Timeline Options - Release tab](../timelines/timeline-options.md#release-tab), which may make playbacks behave differently to how they are programmed. A warning is displayed in the system prompt area to show you what release settings are in effect.
+
+### Auto Simplify
+
+Normally while you are recording, the \[Auto Simplify\] option is enabled. When you stop the recording, this will simplify your fader movements to a few trigger points with linear fades rather than a large number of points which are difficult to edit. If you really need to capture an exotic fader movement you can turn this option off, but to help keep your timeline easy to edit you should leave it on where you can.
+
+Trigger with multiple points during Live Record:
+
+![Timeline before auto simplify](/docs/images/Timeline-Live-Record.png)
+
+The same trigger after Auto Simplify:
+
+![Timeline after auto simplify](/docs/images/Timeline-Live-Record-Simplified.png)
+
+- When you select a trigger (by clicking on it), there are also context menu options in the \{Tools\} context menu to \{Simplify Selected Triggers\} and \{Smooth Selected Triggers\}. The Smooth option will smoothly fade between selected triggers using the fade time of the following trigger.
+
+Adding manual triggers
+-----------------
+
+If you are programming a show where you have been given a list of timecodes for each cue point, you can enter the triggers manually at those specific times.
+
+1.  Press \<Record\>, \[Timeline\], then the **Select button** of the fader where you want to store the timeline. The Timeline View window will automatically open.
+
+2.  Press the \{+\} button at the bottom of the timeline tracks.
+
+    ![Timeline add trigger window](/docs/images/Timeline-Add-Item.png)
+
+3. Select the type of trigger you want to add.
+    - **New Playback** will create a playback from the current programmer contents and trigger it at 100% for 2 seconds then off.
+    - **Existing Playback** lets you select a playback you've programmed previously, and inserts it at 100% for 2 seconds then off.
+    - **Set Level** inserts a trigger to set an existing playback to a specified level.
+    - See the [Triggers](../timelines.md#triggers) section for details of the other types of trigger.
+
+4. If you are using an existing playback, press the **Select** button of the playback you want to fire. (You can skip step 3 and just select a playback straight after pressing the \{+\} button, Titan will behave as if you pressed **Existing Playback**.)
+
+5. Set the time you want the trigger to fire at. There are several ways to do this:
+    - Type in the time using the \[Reference at hh:mm:ss.ttt \] softkey and press \<Enter\>. Use the \<.\> button as a separator, for example typing "1 . 05" will enter 00:01:05:000 or "2 . 2 . 20" will enter 02:02:20.00.
```

Comment:
We would normally say the format is `hh:mm:ss.fff` rather than `ttt`. (f for frame)